client/config/toml.go

```diff
@@ -0,0 +1,88 @@
+package config
+
+import (
+	"bytes"
+	"os"
+	"path"
+	"text/template"
+
+	"github.com/spf13/viper"
+	tmos "github.com/tendermint/tendermint/libs/os"
+)
+
+const defaultConfigTemplate = `# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+
+chain-id = "{{ .ClientConfig.ChainID }}"
+keyring-backend = "{{ .ClientConfig.KeyringBackend }}"
+output = "{{ .ClientConfig.Output }}"
+node = "{{ .ClientConfig.Node }}"
+broadcast-mode = "{{ .ClientConfig.BroadcastMode }}"
+trace = "{{ .ClientConfig.Trace }}"
+`
+
+// InitConfigTemplate initiates config template that will be used in
+// WriteConfigFile
+func InitConfigTemplate() *template.Template {
+	tmpl := template.New("clientConfigFileTemplate")
+	configTemplate, err := tmpl.Parse(defaultConfigTemplate)
+	if err != nil {
+		panic(err)
```

Comment:
Maybe we could change the function to return an error?

Reply:
Good point. I was also thinking about it.